curl_cffi/curl.py

```diff
@@ -310,7 +321,7 @@ def clean_after_perform(self, clear_headers: bool = True) -> None:
                 lib.curl_slist_free_all(self._proxy_headers)
             self._proxy_headers = ffi.NULL
 
-    def duphandle(self) -> "Curl":
+    def duphandle(self) -> Curl:
```

Comment:
What about `typing_extensions.Self`, it seems more consistent.

Reply:
This was just autoformatted, but yeah makes sense

Reply:
Since it returns a different Curl object, a Self annotation doesn't work here. Apparently Self is only for the same Curl object.

Reply:
Understood.